anitya/tests/base.py

```diff
@@ -1,6 +1,6 @@
 # -*- coding: utf-8 -*-
 #
-# Copyright © 2014  Red Hat, Inc.
+# Copyright © 2017  Red Hat, Inc.
```

Comment:
I think this is the result of an odd diff where I made the new base.py file, then started moving stuff out of __init__.py. In the end I moved pretty much everything so git thinks (fairly, I suppose) I renamed the file and made a new __init__.py. I will adjust it.